pkg/apis/crd/v1alpha2/types.go

```diff
@@ -261,16 +261,6 @@ type ExternalIPPoolSpec struct {
 	NodeSelector metav1.LabelSelector `json:"nodeSelector"`
 }
 
-// IPRange is a set of contiguous IP addresses, represented by a CIDR or a pair of start and end IPs.
```

Comment:
any particular reason to move this down? IPRange is also used above in ExternalIPPoolSpec

Reply:
Just to visualize it as "shared" resource (it was between two `External` blocks). Probably not a good reason to introduce extra diff - I moved it back.